src/main/java/model/figure/Circle.java

```diff
@@ -0,0 +1,23 @@
+package model.figure;
+
+import model.Figure;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public int calculateArea() {
+        return (int) (Math.PI * (radius * radius));
+    }
+
+    @Override
+    public void print() {
+        System.out.println("Figure: Circle, area:  " + calculateArea()
+                + " sq. units, radius: " + radius + " units, color: " + color);
```

Comment:
The 'print' method directly prints to the console. This limits the flexibility of the class. Consider using a method that returns the string representation instead, which can then be printed outside the class if needed.